src/services/user/services/AdminUsers.js

```diff
@@ -169,6 +169,13 @@ class AdminUsers {
 
 	async patch(_id, _data, _params) {
 		if (!_id) throw new BadRequest('id is required');
+
+		const currentUser = await getCurrentUserInfo(_params.account.userId.toString());
+		await this.checkIfExternallyManaged(currentUser.schoolId);
+
+		const userToPatch = await getCurrentUserInfo(_id);
+		await this.allowOnlyInternalChanges(currentUser, userToPatch, 'edit');
+
```

Comment:
limit the change to school id instead of requesting them all, is much more performantt

Reply:
this here is the much importants